### ISSUE TITLE
Use task specific test sets from data-split rather than manual splitting of the training set.

### ISSUE THREAD

**Issue by rrenaud** [Activated-AI/multitokenizer-lm#2]
The current implementation splits up the train set to make a test set, which is redundant.
